prow/cluster/jobs/rh-ibm-synergy/multicloud-operators-cluster-controller/rh-ibm-synergy.multicloud-operators-cluster-controller.master.yaml

```diff
@@ -25,7 +25,7 @@ presubmits:
       - command:
         - make
         - build
```

Comment:
Please change command to

      - command:
        - entrypoint
        - make
        - build